jetson/power_logging/model/model_utils.py

```diff
@@ -0,0 +1,46 @@
+import torch
+from typing import Any
+
+def load_model(model_name: str, model_repo: str) -> Any:
+    """Load model from Pytorch Hub.
+
+    Args:
+        model_name: Name of model.
+            It should be same as that in Pytorch Hub.
+
+    Raises:
+        ValueError: If loading model fails from PyTorch Hub
+
+    Returns:
+        PyTorch model
+    """
+    try:
+        return torch.hub.load(model_repo, model_name, pretrained=True)
+    except:
+        raise ValueError(
+            f"Model name: {model_name} is most likely incorrect. "
+            "Please refer https://pytorch.org/hub/ to get model name."
+        )
```

Comment:
I suggest we use the version of this function that we've already used for data collection. This one for instance won't work for lenet

```suggestion
    if model_name == "lenet":
        return LeNet()
    if model_name == "fcn_resnet50":
        return torch.hub.load(model_repo, model_name, pretrained=True)
    try:
        return torch.hub.load(model_repo, model_name)
    except:
        raise ValueError(
            f"Model name: {model_name} is most likely incorrect. "
            "Please refer https://pytorch.org/hub/ to get model name."
        )
```